provider/aws/sia-fargate/sia-fargate.spec

```diff
@@ -0,0 +1,61 @@
+Name:           sia-fargate
+Version:        %{PACKAGE_VERSION}
+Release:        %{RELEASE}.el%{CENTOS_VERSION}
+Summary:        Athenz Service Identity Agent (SIA) for AWS Fargate
+
+Group:          System Environment/Daemons
+License:        Apache 2.0
+URL:            https://www.athenz.io/
+BuildRoot:      %{SOURCEURL0}/rpm/BUILD/%{name}-%{version}-%{release}
+
+%if 0%{?CENTOS_VERSION} == 7
```

Comment:
let's clean up the spec and remove support for centos 6. With the clean up we'll correctly support centos 8